docs/content/guides/dagster/local_to_prod_tutorial.mdx

```diff
@@ -0,0 +1,393 @@
+    return items[items["type"] == "comment"]
+
+
+@asset
+def stories(items: pd.DataFrame) -> pd.DataFrame:
+    """Stories from the Hacker News API."""
+    return items[items["type"] == "story"]
+```
+
+We’ll also need to write the resource to fetch data from Hacker News
+
+<Note>
+
+When you’re in the beginning development stages, you may find it easier to write the code to fetch data directly in your assets or ops. This is great for initial development, but eventually you may want to refactor your code to use resources to support things like testing. In this case we’re going to jump right into defining a resource because we know we’ll want to write a mock resource shortly.
+
+</Note>
+
+```python file=/guides/dagster/local_to_prod_tutorial/resources/resources_v1.py startafter=start_resource endbefore=end_resource
+# resources.py
+
+class HNAPIClient:
+    """
+    Hacker News client that fetches live data
+    """
+
+    @property
+    def item_field_names(self):
+        return [
+            "id",
+            "parent",
+            "time",
+            "type",
+            "by",
+            "text",
+            "kids",
+            "score",
+            "title",
+            "descendants",
+            "url",
+        ]
+
+    def fetch_item_by_id(self, item_id: int) -> Optional[Dict[str, Any]]:
+        """Fetches a single item from the Hacker News API by item id."""
+
+        item_url = f"https://hacker-news.firebaseio.com/v0/item/{item_id}.json"
+        item = requests.get(item_url, timeout=5).json()
+        return item
+
+    def fetch_max_item_id(self) -> int:
+        return requests.get("https://hacker-news.firebaseio.com/v0/maxitem.json", timeout=5).json()
+
+@resource
+def hn_api_client():
+    return HNAPIClient()
+```
+
+Now we can add these assets to our repository and materialize them via Dagit as part of our local development workflow
+
+```python file=/guides/dagster/local_to_prod_tutorial/repository/repository_v1.py startafter=start endbefore=end
+# repository.py
+
+@repository
+def repo():
+    return [
+        *with_resources(
+            [items, comments, stories],
+            resource_defs={"hn_client": hn_api_client, "io_manager": snowflake_io_manager}
+        )
+    ]
+```
+
+When we materialize the assets in Dagit we can add configuration for the `snowflake_io_manager` in the launchpad or we can use the `configured` API and update our repository to look like this:
+
+```python file=/guides/dagster/local_to_prod_tutorial/repository/repository_v2.py startafter=start endbefore=end
+# repository.py
+
+@repository
+def repo():
+    resource_defs = {
+        "hn_client": hn_api_client,
+        "io_manager": snowflake_io_manager.configured(
+            {
+                "account": "abc1234.us-east-1",
+                "user": "me@company.com",
+                "password": "my_super_secret_password",
+                "database": "SANDBOX",
+                "schema": "JAMIE"
```

Comment:
TODO - potentially change the schema here. this would involve actually making a new schema in snowflake so that the included screenshot shows the correct name